platform/ios/src/MGLMapView.mm

```diff
@@ -4106,7 +4106,7 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             // The first selection in the cycle should be the one nearest to the
             // tap.
             CLLocationCoordinate2D currentCoordinate = [self convertPoint:point toCoordinateFromView:self];
-            std::sort(nearbyAnnotations.begin(), nearbyAnnotations.end(), [&](const MGLAnnotationTag tagA, const MGLAnnotationTag tagB) {
+            std::sort(_annotationsNearbyLastTap.begin(), _annotationsNearbyLastTap.end(), [&](const MGLAnnotationTag tagA, const MGLAnnotationTag tagB) {
                 CLLocationCoordinate2D coordinateA = [[self annotationWithTag:tagA] coordinate];
```

Comment:
This change is redundant, because of the `if`, but I prefer the clarity of using _annotationsNearbyLastTap. However...